org.eclipse.lsp4xml.extensions.maven/src/main/java/org/eclipse/lsp4xml/extensions/maven/Utils.java

```diff
@@ -0,0 +1,117 @@
+package org.eclipse.lsp4xml.extensions.maven;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.apache.maven.model.Plugin;
+import org.apache.maven.model.Repository;
+import org.apache.maven.plugin.InvalidPluginDescriptorException;
+import org.apache.maven.plugin.MavenPluginManager;
+import org.apache.maven.plugin.PluginDescriptorParsingException;
+import org.apache.maven.plugin.PluginResolutionException;
+import org.apache.maven.plugin.descriptor.MojoDescriptor;
+import org.apache.maven.plugin.descriptor.Parameter;
+import org.apache.maven.plugin.descriptor.PluginDescriptor;
+import org.apache.maven.project.MavenProject;
+import org.eclipse.aether.repository.RemoteRepository;
+import org.eclipse.aether.repository.RemoteRepository.Builder;
+import org.eclipse.lsp4j.MarkupContent;
+import org.eclipse.lsp4xml.dom.DOMNode;
+import org.eclipse.lsp4xml.services.extensions.IPositionRequest;
+
+public class Utils {
+
+	public static MarkupContent parameterToHoverMarkup(Parameter parameter) {
```

Comment:
should maybe be renamed to `parameterToMarkup`

Reply:
It doesn't seem to be specifically for Hover, but also for completion, so I suggest we rename this method to `getMarkupDescription(Parameter parameter)`. I don't think we need to keep the "parameterTo" in the method name as it's a single argument method so it's easy enough to get what it does.